test/index.spec.ts

```diff
@@ -120,6 +120,13 @@ describe('Public API', () => {
       expect(await fs.readFile(dtsPath, 'utf8')).toContain('declare namespace Electron');
     });
 
+    it('should work default platform/arch', async () => {
```

Comment:
"should work" until someone tries to install, for example, Electron 5 on Linux/mips64el :smile:

Reply:
😢